README.adoc

```diff
@@ -161,3 +180,15 @@ the creation of a new active wallet.
 creation of a new active wallet regardless of age.
 |Yes
 |`25 BTC`
+
+4+s|Sweeping
+
+|depositTreasuryFeeDivisor
+|The deposit fee divisor of one BTC to take as a treasury fee.
+|Yes
+|`2000 => 1/2000 = 5 BPS`
+
+|depositTxMaxFee
+|The max amount of satoshis the wallet is allowed to pay to miners.
```

Comment:
Worth noting this number is per single deposit.

`movedFundsSweepTxMaxTotalFee`, for example, is for the whole move funds transaction.

Reply:
[a26f56b](https://github.com/keep-network/tbtc-v2/pull/332/commits/a26f56b4b0c0f07e4e21af2ef668f620e6c19fb6)